cdc/sink/common/common.go

```diff
@@ -14,21 +14,16 @@
 package common
 
 import (
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
 	"sort"
-	"sync"
-	"sync/atomic"
 
-	"github.com/pingcap/log"
 	"github.com/pingcap/ticdc/cdc/model"
-	"github.com/pingcap/ticdc/pkg/filter"
-	"go.uber.org/zap"
 )
 
-// UnresolvedTxnCache caches unresolved txns
+// UnresolvedTxnCache caches unresolved txns, not thread safe
 type UnresolvedTxnCache struct {
-	unresolvedTxnsMu sync.Mutex
```

Comment:
Why did you remove this?

Reply:
I don't think UnresolvedTxnCache should be thread safe, it's duty should only be txn generator.